src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.spec.ts

```diff
@@ -28,4 +36,89 @@ describe('InstructorHomePageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with default fields', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with one course without feedback session', () => {
+    const instructorName: string = '';
+    const courseTabModels: any = {
+      course: {
+        courseId: 'CS3281',
+        courseName: 'Thematic Systems',
+        creationDate: '26 Feb 23:59 PM',
```

Comment:
This isn't actually rendered anywhere in the HTML. Target the values you are inserting to things being rendered, like the timestamp for feedback sessions. You don't have to change this one, just a heads up, since it's extra work to add things that won't be displayed anyway.